images/nginx-slim/Makefile

```diff
@@ -1,14 +1,66 @@
+# Copyright 2017 The Kubernetes Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
 all: push
 
 # 0.0.0 shouldn't clobber any released builds
-TAG = 0.15
-PREFIX = gcr.io/google_containers/nginx-slim
+TAG = 0.16
+REGISTRY = gcr.io/google_containers
+ARCH ?= $(shell go env GOARCH)
+QEMUVERSION=v2.7.0
+
+IMGNAME = nginx-slim
+IMAGE = $(REGISTRY)/$(IMGNAME)
+MULTI_ARCH_IMG = $(IMAGE)-$(ARCH)
+
+# Set default base image dynamically for each arch
+BASEIMAGE?=gcr.io/google_containers/ubuntu-slim-$(ARCH):0.9
+
+ifeq ($(ARCH),ppc64le)
```

Comment:
@mkumatag by default libaio is not supported. 
You need to remove `--with-file-aio` if `$ARCH != amd64`